projects/1.18/assets/extendedexchange/extendedexchange/lang/zh_cn.json

```
@@ -0,0 +1,131 @@
+{
+  "block.extendedexchange.alchemy_table": "炼金术桌",
+  "block.extendedexchange.basic_collector": "基础能量收集器1型",
+  "block.extendedexchange.basic_power_flower": "基础力量花盆栽罐1型",
+  "block.extendedexchange.basic_relay": "基础反物质继电器1型",
+  "block.extendedexchange.blue_collector": "蓝物质能量收集器8型",
+  "block.extendedexchange.blue_power_flower": "蓝物质力量花盆栽罐8型",
+  "block.extendedexchange.blue_relay": "反蓝物质继电器8型",
```

```suggestion
  "block.extendedexchange.blue_relay": "蓝反物质继电器8型",
```
我的理解：原文的`MK8`是对`反物质继电器`这种整体进行的等级描述，`蓝物质`是物质的一种升级，和`MK8`属于同一等级。即`蓝`=`MK8`。
同时，`反物质`是一个既有名词，就像`蓝物质`一样应该译作`蓝反物质`或`蓝色反物质`（虽然和ProjectE的传统译名风格相悖），而非`反蓝物质`。
此外MK1的翻译也是`基础反物质`而非`反基础物质`。